src/components/data-table/column-sorting/column-sort.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
It looks like it _could be_ but I don't think it should be.  We don't have `.d.ts` files anywhere else except at the root.